bin/kapitan

```diff
@@ -1,4 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
We use this file for local testing sometimes. Any reason to remove it?

Reply:
Actually not sure what happened here. I must of messed up the most recent rebase I did. I'll update the PR to not remove this file.